epan/dissectors/packet-ipp.c

```diff
@@ -428,11 +428,21 @@ dissect_ipp(tvbuff_t *tvb, packet_info *pinfo, proto_tree *tree, void* data)
     operation_status = tvb_get_ntohs(tvb, 2);
     request_id       = tvb_get_ntohl(tvb, 4);
 
-    col_set_str(pinfo->cinfo, COL_PROTOCOL, "IPP");
-    if (is_request)
-        col_add_fstr(pinfo->cinfo, COL_INFO, "IPP Request (%s)", val_to_str(operation_status, operation_vals, "0x%04x"));
-    else
-        col_add_fstr(pinfo->cinfo, COL_INFO, "IPP Response (%s)", val_to_str(operation_status, status_vals, "0x%04x"));
+    if(proto_is_frame_protocol(pinfo->layers, "ippusb")){
```

Comment:
>if( [](start = 4, length = 3)

Follow the same coding style as the rest of the file, e.g.:

"if () {"   note the spaces
and
"} else {"